mediatek: filogic: fix led setup on zyxel-ex5601-t0

gingernut1978 opened the pull request:
Fixes device led setup

Djfe commented:
You need to adjust your local git config.
`git config --local --add user.name "Andrew Sim"`

Alternatively run this as global instead of local, if you want to change the config for everything and not just your local fork of openwrt. (this repo)
`git config --global --add user.name "Andrew Sim"`

`git config --list` shows every config entry (you can limit it further with parameters like `--local` or `--global`

gingernut1978 committed:
mediatek: filogic: rename eth1 to wan in led setup for zyxel-ex5601-t0

the eth1 interface was renamed to wan so also reflect that change in
the leds setup script

Fixes: f26260c7e751 ("mediatek: filogic: Add label wan and cpu for Zyxel EX5601-T0")

Signed-off-by: Andrew Sim <andrewsimz@gmail.com>
Link: https://github.com/openwrt/openwrt/pull/20120
Signed-off-by: Nick Hainke <vincent@systemli.org>

gingernut1978 commented:
Looks like I finally got there

Djfe commented:
@abajk can you approve this please? :)

gingernut1978 commented:
> Please add "Fixes:" Tag referencing the commit that made the change.
> 
> ```
> Fixes: f26260c7e751 ("mediatek: filogic: Add label wan and cpu for Zyxel EX5601-T0")
> ```

Done

PolynomialDivision commented:
Please use the kernel style while doing a Fixes-Tag:
https://www.kernel.org/doc/html/v6.17/process/submitting-patches.html

PolynomialDivision commented:
> Please add "Fixes:" Tag referencing the commit that made the change.
> 
> ```
> Fixes: f26260c7e751 ("mediatek: filogic: Add label wan and cpu for Zyxel EX5601-T0")
> ```

I'm sorry, but the " are missing. Can't you just copy the line I wrote here?

gingernut1978 commented:
@PolynomialDivision Like that?

Thanks for being patient

edit: think I got there in the end :)
